tests/service/views/test_dataset_views.py

```diff
@@ -222,6 +222,78 @@ def test_create_dataset_with_images(svc_client_with_repo):
     assert img2["content_url"].endswith("/2.png")
 
 
+@pytest.mark.service
+@pytest.mark.integration
+@flaky(max_runs=10, min_passes=1)
+def test_create_dataset_with_image_download(svc_client_with_repo):
+    """Create a new dataset with metadata."""
+    svc_client, headers, project_id, _ = svc_client_with_repo
+
+    payload = {
+        "project_id": project_id,
+        "name": "{0}".format(uuid.uuid4().hex),
```

Comment:
```suggestion
        "name": f"{uuid.uuid4().hex}",
```

Reply:
Actually, 
```
>>> a = uuid.uuid4().hex
>>> a == "{0}".format(a)
True
```

Why do we even have that in code